<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangedFilesDialog</class>
 <widget class="QDialog" name="ChangedFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changed files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Changed files:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonReload">
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset theme="cancel"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStashChanges">
       <property name="text">
        <string>Stash changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCommitPush">
       <property name="text">
        <string>Commit/push</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="cancel"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>pushButtonReload</tabstop>
  <tabstop>pushButtonStashChanges</tabstop>
  <tabstop>pushButtonCommitPush</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ChangedFilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
